pkg/fs/realfs.go

```diff
@@ -43,8 +44,32 @@ func (realFS) Open(name string) (File, error) { return os.Open(name) }
 // Stat delegates to os.Stat.
 func (realFS) Stat(name string) (os.FileInfo, error) { return os.Stat(name) }
 
-// ReadFile delegates to ioutil.ReadFile.
-func (realFS) ReadFile(name string) ([]byte, error) { return ioutil.ReadFile(name) }
+var separator = []byte(`
+---
+`)
+
+// ReadFile use glob to find the matching files and then read content from all of them
+func (realFS) ReadFile(name string) ([]byte, error) {
+	output := []byte{}
```

Comment:
How does this compare to using a ByteBuffer?

Reply:
Maybe `bytes.ByteBuffer` is better. I will compare them

Reply:
Cool, I am curious to know the answer :)

Reply:
I used `[][]byte`, so that I have an easy way to Join content from different files with a separator. ` bytes.ByteBuffer` handles bytes manipulation better, such as truncate certain number of bytes.